.github/pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+# Changes
```

Comment:
a.) According to https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository it looks like it can be just about anywhere in the repository structure.
b) Any files present in a repository such as shipwright-io/build will override any present in this .github repository. Once this repository is merged, we would remove certain files from the other repositories so that everything is in sync.

Reply:
ok, agree!